bricks/dart_frog_new/hooks/lib/src/route_to_path.dart

```diff
@@ -14,19 +16,23 @@ String routeToPath(
   String route, {
   bool preferIndex = false,
   String preamble = 'routes',
+  path.Context? pathContext,
 }) {
+  final context = pathContext ?? path.context;
+  final separator = context.separator;
+
   if (route == '/') {
-    return '$preamble/index.dart';
+    return '$preamble${separator}index.dart';
   }
 
-  final path =
-      route.split('/').where((element) => element.isNotEmpty).join('/');
+  final p =
```

Comment:
Why are we renaming`path` to `p`? I think the previous name is more descriptive?

Reply:
This conflicts with the import.